includes/components/class-attribute.php

```diff
@@ -776,6 +776,10 @@ public function add_model_fields( $fields, $object ) {
 			}
 		}
 
+		if ( ! $category_ids ) {
```

Comment:
Doesn't seem to be ok, it checks categories assigned to a listing (fetched above), not if categories exist at all

Reply:
Please check how we do this for listings, probably via a callback in the Listing component, unsetting the field in the form and making it optional in the model. It's better to unify this, then the issue will be resolved for listings, vendors, requests, etc. and any future attribute-supported models.